strum/src/lib.rs

```diff
@@ -152,24 +152,22 @@ pub trait EnumMessage {
 ///     #[strum(props(Teacher="Ms.Frizzle", Room="201"))]
 ///     History,
 ///     #[strum(props(Teacher="Mr.Smith"))]
-///     #[strum(props(Room="103"))]
+///     #[strum(props(Room=103))]
 ///     Mathematics,
 ///     #[strum(props(Time="2:30"))]
 ///     Science,
 /// }
 ///
 /// let history = Class::History;
 /// assert_eq!("Ms.Frizzle", history.get_str("Teacher").unwrap());
+/// let maths = Class::Mathematics;
+/// assert_eq!(103, maths.get_int("Room").unwrap());
 /// ```
 pub trait EnumProperty {
     fn get_str(&self, prop: &str) -> Option<&'static str>;
-    fn get_int(&self, _prop: &str) -> Option<usize> {
-        Option::None
-    }
+    fn get_int(&self, prop: &str) -> Option<usize>;
```

Comment:
`get_usize` is probably a more future-proof name

Reply:
I'm agree and I'm happy to make that change, but I also thought it'd be worth mentioning that while it'll be more future proof, it's also technically a breaking change. I'd be _immensely_ surprised if any real code got broken, but I just wanted to check you're ok with that.

Reply:
I jinxed myself by saying earlier nothing would break, because there seems to be an odd dependency between `strum_macros` and a previous version of the main package. I've made the change and updated the Cargo.toml to use the local version, which solved the issue.

Reply:
Hey @ejmount, sorry for the delay getting back to you. I've been out of town for a few weeks. I missed that I had stubbed that method out at some point in the past. If you're still interested in completing this, here's what we can do. I can publish a new version of strum (not macros) with an updated trait definition then we can rebase your changes onto the new version.

Reply:
Yeah, it [sounds like](https://github.com/rust-lang/cargo/issues/1565) that's still not supported, so it'd be great if you could publish a new version to make that update.